mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -208,8 +195,7 @@ class AFTSurvivalRegression @Since("1.6.0") (@Since("1.6.0") override val uid: S
   }
 
   @Since("2.0.0")
```

Comment:
We change from `fit` to `train`. Strictly it is not `since 2.0.0`.

Reply:
Changed to 3.0.0. Thanks!